dexter/articles.py

```diff
@@ -22,13 +25,44 @@
 
 @app.route('/articles/<id>')
```

Comment:
We chatted about showing this analysis under `/articles/<id>/fdi`. The benefit is that users with both roles (eg. William and Boots) can still look at both types of analysis. Currently this will force them to only view FDI analysis.